projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -97,6 +98,13 @@ class Contact_Form_Plugin {
 		'feedback_id'             => '',
 	);
 
+	/**
+	 * GDrive export nonce field name
+	 *
+	 * @var string The nonce field name for GDrive export.
+	 */
+	private $export_nonce_field_gdrive = 'feedback_export_nonce_gdrive';
+
```

Comment:
I wonder if it makes any sense to move all the Google drive logic to the [Google_Drive](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/forms/src/service/class-google-drive.php) class that was created in August? That's in `forms/src/services/.`